b2share/modules/deposit/api.py

```diff
@@ -224,6 +226,25 @@ def submit(self, commit=True):
         if commit:
             self.commit()
 
+    def validate_before_publishing(self):
+        """Common sense validations before submitting. These would be also
```

Comment:
I know we replace some of the messages, what I meant is "can we do a special case for this error when we replace the error message"? Like using a regular expression on the error message to catch specifically the one we want to translate.